lib/Watcher.js

```diff
@@ -43,7 +43,7 @@ function Watcher(options) {
             ignored: normalizedIgnorePatterns,
             cwd: dir,
             persistent: (options.persistent !== false),
-            usePolling: false,
+            usePolling: (options.usePolling !== false),
```

Comment:
Using the ternary operator would be more appropriate in line with the pass through option in browser-refresh https://github.com/patrick-steele-idem/browser-refresh/commit/af6510f555a0688e3b6f5856bb77ab140cd8d232. 

Main idea was to keep the options defaulting to the same values, unless the configuration option was passed.